server/events/command_runner.go

```diff
@@ -179,7 +179,11 @@ func (c *DefaultCommandRunner) RunAutoplanCommand(logger logging.SimpleLogging,
 	err = c.PreWorkflowHooksCommandRunner.RunPreHooks(ctx)
 
 	if err != nil {
-		ctx.Log.Errorf("Error running pre-workflow hooks %s. Proceeding with %s command.", err, command.Plan)
+		ctx.Log.Errorf("Error running pre-workflow hooks %s. Aborting %s command.", err, command.Plan.String())
```

Comment:
can you use my new logger here pls.

Reply:
Also no formatting allowed in logs anymore.

Reply:
Are you talking about `logging.Logger` interface and do I need to thread it into command runner?